cli/src/diff_util.rs

```diff
@@ -567,7 +569,12 @@ pub fn show_color_words_diff(
 ) -> Result<(), DiffRenderError> {
     let mut diff_stream = materialized_diff_stream(store, tree_diff);
     async {
-        while let Some((path, diff)) = diff_stream.next().await {
+        while let Some(MaterializedTreeDiffEntry {
+            source: _,
```

Comment:
nit: add TODO about using this value (applies throughout this commit)

Reply:
done